account-kit/smart-contracts/src/ma-v2/client/client.test.ts

```diff
@@ -94,6 +99,19 @@ describe("MA v2 Tests", async () => {
     accounts.unfundedAccountOwner
   );
 
+  const webAuthnCredential = await createWebAuthnCredential({ name: "owner" });
+
+  const webAuthnCredentialBackUp = await createWebAuthnCredential({
+    name: "backUp",
+  });
+
+  const webAuthnSigningMethods: SmartAccountSigner = new WebAuthnSigningMethods(
```

Comment:
I think I am mistakenly treating this like an instance of `SmartAccountSigner` when this is a standalone collection of webauthn signing methods, so it would get access to the WebAuthnCredentials via the account

Reply:
I left comments in the spec about this. It's not a smart account signer, it's what's used to create the various `sign*` methods on the account itself. the account doesn't take a signer as input anymore, and just takes in the credential